packages/link/src/react/index.tsx

```diff
@@ -1,11 +1,13 @@
-import { css } from 'glamor'
 import { useTheme } from '@pluralsight/ps-design-system-theme'
 import { appearances } from '../vars'
-import { RefForwardingComponent } from '@pluralsight/ps-design-system-util'
+import {
+  RefForwardingComponent,
+  classNames
+} from '@pluralsight/ps-design-system-util'
 
 import React, { HTMLAttributes } from 'react'
 
-import stylesheet from '../css'
+import '../css/index.css'
```

Comment:
I want to hear more of your perspective in this POC of how the developer experience has changed for users of the DS.  

Specifically, a component used to be npm installable and then usable without further bundler configuration. Has that changed? If so, how?